src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.Maui.IntegrationTests/MauiTestHelper.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Forms.IntegrationTests.Common;
+using Xunit;
+
+namespace System.Windows.Forms.Maui.IntegrationTests
+{
+    /// <summary>
+    /// This class performs common operations for maui xUnit tests.
+    /// </summary>
+    public static class MauiTestHelper
+    {
+        /// <summary>
+        /// The test runner
+        /// </summary>
+        private static readonly MauiTestRunner s_testRunner = new MauiTestRunner();
+
+        /// <summary>
+        /// The test results, indexed by project name
+        /// </summary>
+        private static readonly Dictionary<string, TestResult> s_testResults = new Dictionary<string, TestResult>();
+
+        /// <summary>
+        /// Run the maui test for the specified project name and store the results
+        /// </summary>
+        /// <param name="projectName">The name of the project to run</param>
+        public static void RunMauiTest(string projectName)
```

Comment:
With the above, this can be turned into a `private`.
```suggestion
        public static void RunTests(string projectName)
```